lib/src/widgets/utils/ds_card.widget.dart

```diff
@@ -258,6 +258,7 @@ class DSCard extends StatelessWidget {
         shouldAuthenticate: shouldAuthenticate,
       );
     } else if (media.type.contains('video')) {
+      final fileName = media.title?.split('.').first;
```

Comment:
To get the filename without the extension of the file, you can use: 

Example: "Erro.teste.teste2.mp4"

`media.title?.substring(0, media.title?.lastIndexOf('.'))`.  --> "Erro.teste.teste2"
`media.title?.split('.').first` --> "Erro"